app.ts

```diff
@@ -0,0 +1,319 @@
+let fromNumber = 0;
+let recordNumberTotal: number;
+let count = 0;
+let timeout = 0;
+
+window.onload = () => {
+	createNavigation();
+	recordCount();
+	headingRowCreation();
+	fromNumber = 0;
+	new pageNavigation();
+};
+
+function checkResponseError(response: Response) {
+	if (!response.ok) {
+		throw Error(response.statusText);
+	}
+	return response;
+}
+
+function debounce(func: any, delay: number) {
+	return function () {
+		clearTimeout(timeout);
+
+		timeout = setTimeout(() => {
+			func();
+		}, delay);
+	};
+}
+
+function createNavigation() {
+	let recordNav: HTMLElement | null = document.getElementById("record-navigation-container"); // Navigation area
+	if (recordNav !== null) {
+		recordNav.innerHTML = `
+		<div id="navigation-btns">
+			<button value="first" id="first-page-btn">First Page</button>
+			<button value="previous" id="previous-records-btn">Previous</button>
+			<button value="next" id="next-records-btn">Next</button>
+			<button value="last" id="last-page-btn">Last Page</button>
+			<button id="confirmation-btn">Get Record</button>
+		</div>
+		<div class="current-page-container">
+			<p id="current-page"></p>
+		</div>
+		`;
+	}
+}
+
+function getRecords(fromNumber: number, toNumber: number): Promise<void> {
+	return fetch(`http://localhost:2050/records?from=${fromNumber}&to=${toNumber}`, {
+		method: "GET",
+		headers: { "Content-Type": "application/json" },
+	})
+		.then(checkResponseError)
+		.then((response: Response) => response.json())
+		.then((data: string) => {
+			let infoColumns: HTMLElement | null = document.getElementById("info-columns-container"); // Information
+			let currentPage: HTMLElement | null = document.getElementById("current-page");
+
+			if (infoColumns !== null) {
+				(infoColumns as HTMLDivElement).innerHTML = "";
+
+				for (let i of data) {
+					dynamicGrid(i);
+				}
+			}
+
+			if (currentPage !== null) {
+				(currentPage as HTMLParagraphElement).innerHTML = `${fromNumber}  / ${toNumber}.`;
+			}
+		})
+		.catch((error: Error) => {
+			console.log(error);
+		});
+}
+
+function recordSelection() {
+	let recordNav: HTMLElement | null = document.getElementById("record-navigation-container"); // Navigation area
+	if (!recordNav) {
+		alert("The navigation is not working correctly refresh the page");
+	} else {
```

Comment:
Please add a `return` after the alert and remove the else, no need to have your whole function inside the else of an if statement.